R/defaults.R

```diff
@@ -1,3 +1,16 @@
+#' Default method for data validation
+#'
+#' @inheritParams epidist_validate
+#' @param ... Additional arguments for method.
+#' @family defaults
+#' @export
+epidist_validate.default <- function(data, ...) {
+  stop(
+    "No epidist_formula method implemented for the class ", class(data), "\n",
```

Comment:
we might want to say no validate method defined vs formula as there are more than just formula methods you would need. At a later date (i.e. this is another issue) we should add a link how to some kind of documentaiton on how to make a model class

Reply:
This is just a forget to edit not an intentional.